R/template_config.R

```diff
@@ -0,0 +1,47 @@
+#' @export
+format_edge_type <- function(edge_types) {
+  et <- dplyr::bind_rows(lapply(edge_types, function(x) data.frame(value=x[[2]], schema_name=x[[1]])))
+  et |> dplyr::filter(value %in% c("Component", "Filename")) |> 
```

Comment:
Filter only by component to avoid edge case where data model has an attribute that depends on Filename but is not a component. Filter to filename separately then merge the results.